spec.html

```diff
@@ -9473,6 +9466,19 @@ <h1>IntegerIndexedObjectCreate ( _prototype_, _internalSlotsList_ )</h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-integerindexedelementget" aoid="IntegerIndexedElementGet">
+        <h1>IsIntegerIndexValid ( _O_, _index_ )</h1>
+        <p>The abstract operation IsIntegerIndexValid with arguments _O_ and _index_ performs the following steps:</p>
+        <emu-alg>
+          1. Assert: Type(_index_) is Number.
+          1. Assert: _O_ is an Object that has an [[ArrayLength]] internal slot.
+          1. If IsInteger(_index_) is *false*, return *false*.
+          1. If _index_ = *-0*, return *false*.
```

Comment:
```suggestion
          1. If _index_ is *-0*, return *false*.
```
perhaps?

Reply:
There is a well-established precedence for using `=` to check for mathematical values, though *-0* is kind of a corner case. My preference is to keep the `=`.

Reply:
There's 5 cases of `= -0` in the spec right now, and 81 of `is -0`, so precedent seems to land on the side of the suggestion. I'd be happy to clean up the wayward 5.

Reply:
Sure, sounds fine to me.